graphql/query_resolver.go

```diff
@@ -488,7 +488,14 @@ func (r *queryResolver) ViewableProjectRefs(ctx context.Context) ([]*GroupedProj
 		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error getting projects: %v", err.Error()))
 	}
 
-	groupedProjects, err := groupProjects(projects, true)
+	visibleProjects := []model.ProjectRef{}
```

Comment:
Should we move this into groupProjects? I think the other place it's called is also relevant; can add a comment to it or rename groupVisibleProjects